src/chrome/content/rules/ICPDR.org.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Invalid certificate:
+		- icpdr.org, equivalent to www.icpdr.org
+		- danubis.icpdr.org
```

Comment:
`danubis.icpdr.org` WFM

Reply:
Thanks for catching that, no idea what I was doing there 😛